nibabel/optpkg.py

```diff
@@ -115,6 +115,7 @@ def optional_package(name, trip_msg=None, min_version=None):
                     % (name, name, exc))
     pkg = TripWire(trip_msg)
 
+    # TODO dj: no clue why is it needed...
```

Comment:
This allows a module that, for instance, depends on `h5py` to skip its tests if `h5py` is not installed.

Reply:
but you don't want to move this to the tests? i was surprised that this is a part of the main code.
If it's better  to keep it here, I'll just change simply change it to the pytest function

Reply:
This affects doctests, not just tests.